cmd/apps.go

```diff
@@ -276,3 +284,33 @@ func AppTransfer(appID, username string) error {
 
 	return nil
 }
+
+var errNoDomain error
```

Comment:
can we remove this error from package-global scope? if it needs to be package-global, then make it a `struct`